index.js

```diff
@@ -241,25 +269,22 @@ const nextMatch = async matches => {
   }
 
   if (matches > 1) {
-    await nextMatch(--matches);
+    await nextMatch(matches - 1);
```

Comment:
Alright, alright. We don't do anything with the variable after this so we don't need to `--`, I get it. I still like it haha :-/

Reply:
🤷‍♂️